<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtlasEditorUI</class>
 <widget class="QMainWindow" name="AtlasEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Atlas Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionSaveProject"/>
   <addaction name="actionSaveProjectAs"/>
   <addaction name="actionLoadProject"/>
   <addaction name="actionReloadProject"/>
   <addaction name="separator"/>
   <addaction name="actionCellMode"/>
   <addaction name="actionOriginMode"/>
   <addaction name="actionClipMode"/>
   <addaction name="separator"/>
   <addaction name="actionImportAtlas"/>
   <addaction name="actionExportAtlas"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCellMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/atlas/icons/atlas/cellmode.png</normaloff>:/atlas/icons/atlas/cellmode.png</iconset>
   </property>
   <property name="text">
    <string>Cell Mode</string>
   </property>
   <property name="toolTip">
    <string>Cell Mode</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionOriginMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/atlas/icons/atlas/originmode.png</normaloff>:/atlas/icons/atlas/originmode.png</iconset>
   </property>
   <property name="text">
    <string>Origin Mode</string>
   </property>
   <property name="toolTip">
    <string>Origin Mode</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionClipMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/atlas/icons/atlas/clipmode.png</normaloff>:/atlas/icons/atlas/clipmode.png</iconset>
   </property>
   <property name="text">
    <string>Clip Mode</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/new.png</normaloff>:/essentials/icons/essentials/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/save.png</normaloff>:/essentials/icons/essentials/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveProjectAs">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/save_as.png</normaloff>:/essentials/icons/essentials/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/load.png</normaloff>:/essentials/icons/essentials/load.png</iconset>
   </property>
   <property name="text">
    <string>Load Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExportAtlas">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/export.png</normaloff>:/essentials/icons/essentials/export.png</iconset>
   </property>
   <property name="text">
    <string>Export Atlas</string>
   </property>
  </action>
  <action name="actionImportAtlas">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/import.png</normaloff>:/essentials/icons/essentials/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Atlas</string>
   </property>
  </action>
  <action name="actionReloadProject">
   <property name="icon">
    <iconset resource="../elang_gui_icons.qrc">
     <normaloff>:/essentials/icons/essentials/reload.png</normaloff>:/essentials/icons/essentials/reload.png</iconset>
   </property>
   <property name="text">
    <string>Reload Project</string>
   </property>
   <property name="toolTip">
    <string>Reload Project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../elang_gui_icons.qrc"/>
 </resources>
 <connections/>
</ui>
